x-pack/plugin/sql/qa/server/security/build.gradle

```diff
@@ -24,7 +15,9 @@ subprojects {
 
   dependencies {
     testImplementation project(":x-pack:plugin:core")
-    testArtifacts project(path: mainProject.path, configuration: 'testArtifacts')
+    testArtifacts(testArtifact(project(mainProject.path))) {
+      transitive = false
```

Comment:
Why do we have to disable transitive dependencies here? Couldn't we do this on the configuration?

Reply:
true. changed to be done on the configuration